backend/handlers/get_rule_graph.go

```diff
@@ -0,0 +1,69 @@
+package handlers
+
+import (
+	"github.com/Zeus-Labs/ZeusCloud/rules/types"
+	"log"
+	"net/http"
+
+	"github.com/Zeus-Labs/ZeusCloud/rules/attackpath"
+	"github.com/neo4j/neo4j-go-driver/v4/neo4j"
+)
+
+var ruleGraphMap = map[string]types.Rule{
+	"private_serverless_admin_permissions":          attackpath.PrivateServerlessAdmin{},
+	"private_vm_admin_permissions":                  attackpath.PrivateVmAdmin{},
+	"publicly_exposed_vm_admin_permissions":         attackpath.PubliclyExposedVmAdmin{},
+	"publicly_exposed_vm_high_permissions":          attackpath.PubliclyExposedVmHigh{},
+	"publicly_exposed_vm_priv_escalation":           attackpath.PubliclyExposedVmPrivEsc{},
+	"publicly_exposed_serverless_admin_permissions": attackpath.PubliclyExposedServerlessAdmin{},
+	"publicly_exposed_serverless_high_permissions":  attackpath.PubliclyExposedServerlessHigh{},
+	"publicly_exposed_serverless_priv_escalation":   attackpath.PubliclyExposedServerlessPrivEsc{},
+	"third_party_admin_permissions":                 attackpath.ThirdPartyAdmin{},
+	"third_party_high_permissions":                  attackpath.ThirdPartyHigh{},
+}
+
+func GetRuleGraph(driver neo4j.Driver) func(w http.ResponseWriter, r *http.Request) {
+	return func(w http.ResponseWriter, r *http.Request) {
+		log.Printf("Entered Get Rule Graph \n")
+		resourceId := r.URL.Query().Get("resource")
+		rule := r.URL.Query().Get("rule")
```

Comment:
frontend can pass the full rule uid - use that so that we're consistently refferring to the rule the same way everywhere. Otherwise, this introduces a second naming schemes for rules